reconcile/templating/renderer.py

```diff
@@ -245,7 +252,13 @@ class TemplateRendererIntegrationParams(PydanticRunParams):
 
 
 def join_path(base: str, sub: str) -> str:
-    return os.path.join(base, sub.lstrip(APP_INTERFACE_PATH_SEPERATOR))
+    clean_sub = sub.lstrip(APP_INTERFACE_PATH_SEPERATOR)
+    # Handle special case: if targetPath starts with "resources",
+    # write to sibling resources/ directory instead of under data/
+    if clean_sub.startswith("resources/"):
+        base_parent = os.path.dirname(base)  # /foo/bar/data -> /foo/bar
+        return os.path.join(base_parent, clean_sub)
+    return os.path.join(base, clean_sub)
```

Comment:
we should proper support data path and resource path, or just root path instead of treat `resources` as a special case

Reply:
What would you advise here? 

Updating to root path would entail updating all outstanding templates in app-interface but minimal code changes.

Alternatively, the template schema could be updated w/ an optional attribute (default to `data`) to specify root directory?

Reply:
should we also update `--app-interface-data-path` and `APP_INTERFACE_DATA_PATH` to be root path?

Reply:
good call. Updated in [97537b3](https://github.com/app-sre/qontract-reconcile/pull/5153/commits/97537b35912da7ebb3422583559d1b4250ddeb0f)